tachyon/c/zk/air/plonky3/baby_bear_poseidon2_constants.h

```diff
@@ -0,0 +1,14 @@
+#ifndef TACHYON_C_ZK_AIR_PLONKY3_CONSTANTS_H_
+#define TACHYON_C_ZK_AIR_PLONKY3_CONSTANTS_H_
+
+// Poseidon2 constants
+#define TACHYON_PLONKY3_BABY_BEAR_POSEIDON2_WIDTH 16
+#define TACHYON_PLONKY3_BABY_BEAR_POSEIDON2_ALPHA 7
+#define TACHYON_PLONKY3_BABY_BEAR_POSEIDON2_FULL_ROUNDS 8
+#define TACHYON_PLONKY3_BABY_BEAR_POSEIDON2_PARTIAL_ROUNDS 13
+
+#define TACHYON_PLONKY3_BABY_BEAR_POSEIDON2_RATE 8
+#define TACHYON_PLONKY3_BABY_BEAR_POSEIDON2_CHUNK 8
+#define TACHYON_PLONKY3_BABY_BEAR_POSEIDON2_N 2
```

Comment:
If this config is defined by SP1, then a name like `TACHYON_PLONKY3_SP1_POSEIDON2_XX` would be more appropriate, don't you think?

Reply:
If this config depends on F, then we might need to keep BABY_BEAR!

Reply:
If you look at different cases in Plonky3, it appears that the field type does dictate different possible values for these constants as shown [here](https://github.com/Plonky3/Plonky3/blob/main/poseidon2/benches/poseidon2.rs) (note that D = ALPHA). Full rounds and partial rounds are also dependent on width, alpha, and field as stated in this comment in poseidon [here](https://github.com/Plonky3/Plonky3/blob/55832146c86e8e4d246bb9843da17f2159d212a5/poseidon/benches/poseidon.rs#L40)